Makefile

```diff
@@ -95,3 +95,28 @@ chlog-preview: chlog-validate ## Provide a preview of the generated CHANGELOG.md
 chlog-update: chlog-validate ## Creates an update to CHANGELOG.md for a release entry from content in .chloggen
 	$(CHLOGGEN) update --version "[$(VERSION)] - $$(date +'%Y-%m-%d')" || exit 1; \
 	ci_scripts/chloggen-update.sh || exit 1
+
+CERTMANAGER_VERSION ?= 1.10.0
```

Comment:
```suggestion
CERTMANAGER_VERSION ?= $(shell yq eval ".dependencies[] | select(.name == \"cert-manager\") | .version" helm-charts/splunk-otel-collector/Chart.yaml)
```

Reply:
thanks, added to the latest.

Reply:
looks like it's not applied

Reply:
I applied it in the next version, leaving this PR untouched. The change is not enough, I had to change the make target a bit to remove the `v` of the version.